mobify/platform-scaffold
# SFCC default sorting option

Change the default sorting option from blank to `Relevance`.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-147
 ~~**Linked PRs**: (links to corresponding PRs, optional)~~

## Changes
- Default Sorting option: "" -> `Relevance`

## Todos
~~- [ ] (if applicable) analytics events instrumented to measure impact of change~~
~~- [ ] Add a high-level description of your changes to the CHANGELOG.md, including a link to the PR with your changes~~

## How to test-drive this PR
- import SFCC connector
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Go to PLP and verify sorting option
